src/main/java/uk/gov/ch/developer/docs/config/NavBarConfig.java

```diff
@@ -44,14 +44,6 @@ public NavBarModelBuilder getNavBarModelBuilder() {
         documentation.add("How to add an API key", authenticationUrl);
         documentation.add("Developer Guidelines", devGuideURL);
 
-        NavItemList manageAccounts = model
-                .addHeading("Manage account", DisplayRestrictions.USER_REQUIRED);
-        manageAccounts.add("Manage Profile", homeURL);
-        manageAccounts.add("Change Password", homeURL);
-
-        NavItemList help = model.addHeading("Help", DisplayRestrictions.USER_REQUIRED);
```

Comment:
It seems a little of that the help is only restricted to logged in users.

Reply:
This was removed.

Reply:
My mistake, sorry.